roles/installer/tasks/enable_metrics.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Check to make sure provided pvc exists, error loudly if not.  Otherwise, the management pod will just stay in pending state forever.
+- name: Check provided PVC claim exists
+  kubernetes.core.k8s_info:
+    name: "{{ _metrics_pvc_claim }}"
+    kind: PersistentVolumeClaim
+    namespace: "{{ ansible_operator_meta.namespace }}"
+  register: provided_pvc
+  when:
+    - _metrics_pvc_claim | length
+
+- block:
```

Comment:
I don't think we need block statements here.
Also we could probably group the three k8s resources creation in a single k8s apply task + loop

https://github.com/ansible/awx-operator/blob/devel/roles/installer/tasks/resources_configuration.yml#L203-L216